### ISSUE TITLE
Accessibility: MAS40B: Inspect: Localized control type property is wrong for ".tar.gz" link

### ISSUE THREAD

**Issue by wiproaccess** [wiproaccess/vscode-website#21]
#A11yMAS
#MAS40B
#Wipro
#vscode-website

**Testing Environment:**
OS-RS1
OS Name Microsoft -Windows 10 Pro
Version-10.0.14393 Build 14393.607
Browser-Edge
Edge Version-Microsoft Edge 38.14393.607.0

**Repro Steps:**
1. Open [https://code.visualstudio.com/](https://code.visualstudio.com/) in EDGE.
2. Navigate to ".tar.gz" link (with mouse as keyboard is not accessible) available under the Debian, Ubuntu download link inside "Get Started Now" header.
3. Observe the Localized Control Type property in Inspect

Same issue is also observed with the ".zip" link available under the Windows Download link inside "Get Started Now" Header

**_Same issue is also observed with the links:_**
[https://code.visualstudio.com/Download
](https://code.visualstudio.com/Download
)https://code.visualstudio.com/insiders

**Actual Result:**
The Localized Control Type property of the ".tar.gz" link is text

**_Narrator Behaviour:_**
Narrator announces as ".tar.gz"

**Expected Result:**
The Localized Control type property should be link instead of text

**User Impact:**
The visually impaired users will not be able to access the interactive link.

**Screenshots:**
![default tar gz link localized control type property is text](https://cloud.githubusercontent.com/assets/24825441/21709150/bb5de8a6-d404-11e6-84a8-30a11142f998.png)
![localized control type of zip link is text](https://cloud.githubusercontent.com/assets/24825441/21709149/bb5781e6-d404-11e6-81d0-e72a4602ba55.png)

**Comment by chrisdias** [comment]
fixed with https://github.com/Microsoft/vscode-website/commit/6efd7c2db5268d27dd8c690eb8a501420fe345ad

**Comment by wiproaccess** [comment]
The issue is verified hence adding the label 'closed' and closing the bug.
